engine/Default/galactic_post_current.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$db->query('SELECT * FROM galactic_post_paper WHERE online_since IS NOT NULL AND game_id = ' . $db->escapeNumber($player->getGameID()) . ' ORDER BY online_since DESC');
```

Comment:
It'd be worth adding a `LIMIT 1` so that we only fetch the first row rather than all of the matching ones

Reply:
Done!